fetch2/src/main/java/com/tonyodev/fetch2/DownloadManager.java

```diff
@@ -54,6 +54,30 @@ void pause(long id) {
         actionProcessor.processNext();
     }
 
+    void pause(final String groupId) {
```

Comment:
@jacobtabak.The downloadManager can interrupt a download with the reason why it was interrupted. The interrupted download, stops, updates its progress and status base on the reason and terminates. "Queued" for a new request means it will be inserted into the database and start as soon as it has been inserted.